aptos-move/framework/aptos-framework/sources/aptos_governance.move

```diff
@@ -179,11 +179,14 @@ module aptos_framework::aptos_governance {
         if (!exists<GovernanceResponsbility>(@aptos_framework)) {
             move_to(
                 aptos_framework,
-                GovernanceResponsbility { signer_caps: simple_map::create<address, SignerCapability>() }
+                GovernanceResponsbility {
+                    signer_caps: simple_map::create<address, SignerCapability>()
```

Comment:
Might as well remove < address, SignerCapability > as that's not necessary

Reply:
I think we would like to let the linter take care of this? @vineethk

Reply:
Yeah, the formatter should ideally not be modifying code like this (eg., the rust formatter does not remove unnecessary type annotations AFAIK). Among other things, such a functionality would require the code being formatted to be type-safe already, which is not desirable.